### PR TITLE
use rake vendor before running test

### PR DESCRIPTION
**Pull request by purbon**
Fixes travis ci build.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,4 +4,4 @@ cache: bundler
 rvm:
   - jruby-1.7.23
 script: 
-  - bundle exec rspec spec
+  - rake vendor; bundle exec rspec spec
```
Comment (ph) [inline .travis.yml]: I think it should be this.

``` yaml
before_script: bundle exec rake vendor
script: bundle exec rspec spec
```

Comment (purbon) [inline .travis.yml]: sure, must admit I'm not confortable with travis-ci file syntax :+1:

**Comment by purbon** [comment]
@ph updated, lets see if now works properly ... before the PR test failed.

**Comment by ph** [comment]
LGTM

**Pull request closed without merging**